todo.md

```diff
@@ -155,6 +157,7 @@
 
 *   Build custom theme
 *   Reserve space for the full size image on the home page. When the page is loaded, the content won't jump around
+*   Wordpress minify assets https://wordpress.org/plugins/bwp-minify/
```

Comment:
Could you use the markdown link format? E.g., `[WordPress minify assets](https://wordpress.org/plugins/bwp-minify/)`